screenpy_selenium/resolutions/custom_matchers/is_invisible_element.py

```diff
@@ -0,0 +1,46 @@
+"""
+A matcher that matches an invisible element. For example:
+
+    assert_that(driver.find_element_by_id("search"), is_invisible_element())
+"""
+
+
+from typing import Optional
+
+from hamcrest.core.base_matcher import BaseMatcher
+from hamcrest.core.description import Description
+from selenium.webdriver.remote.webelement import WebElement
+
+
+class IsInvisibleElement(BaseMatcher[Optional[WebElement]]):
```

Comment:
If you do wish to keep the `IsInvisible` approach (which, i agree, i would like that), i would prefer it if you modify the existing matcher slightly to allow a boolean to be passed in, and use that in `is_invisible_element`.

(We've been talking about this on Discord, i just want to also have the conversation here so we can look back and remember why we went whichever way we went, some distant future day).

Reply:
I decided against using a single matcher due to SRP.  It felt more explicit to have _nearly_ (but not exactly) duplicate code.  I had to remind myself that part of the purpose of these matchers is to provide the proper logging strings.